### PR TITLE
Implement persistent findings storage - Part 3

### PR DESCRIPTION
**Pull request by andream16**
Part 3 of https://github.com/smithy-security/smithy/pull/492.

---

**How to review these changes**

- Review this PR
- Review https://github.com/smithy-security/smithy/pull/496
- Review https://github.com/smithy-security/saas/pull/806

image

---

- Using migrations & SQLite generation like we did for postgres for consistency

### CODE REVIEW

**Review by dlicheva — APPROVED** [review]

File: sdk/README.md [hunk 1]
```diff
@@ -329,14 +329,20 @@ This is achieved with migrations.
...
 ##### SQLite
 
-TBD.
+Postgresql migrations live in `./component/store/local/sqlc/sqlc/migrations`.
+
+In order to create a new migration, you can follow these steps:
+
+* run `$ make new-sqlite-migration migration_name=my_migration`
+* edit the migration
+* run `$ update-sqlite-migrations-sum` to update `atlas.sum`
```
Comment (dlicheva) [inline sdk/README.md:338]: What is `altlas.sum` and why is it needed? Is that the migration manager?

Comment (andream16) [reply sdk/README.md:338]: It's used by atlas to understand if the generated migrations are valid or they were manually changed

Comment (dlicheva) [reply sdk/README.md:338]: Should that be in the readme?

**Review by dlicheva — APPROVED** [review]

File: sdk/README.md (hunk 1 above)
Comment (andream16) [reply sdk/README.md:338]: Can add in the next PR!

**Pull request merged**